src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -128,6 +130,25 @@ const NewFormRuleDialog: React.FC<Props> = ({
     },
   });
 
+  // This would be better handled with real form validation
```

Comment:
I feel like it would be better to use rhf for forms that aren't "dynamic" but that would require some refactoring.